build_definitions/crypt_blowfish.py

```diff
@@ -27,11 +27,16 @@ def __init__(self):
                 'https://github.com/YugaByte/crypt_blowfish/archive/{0}.tar.gz',
                 BUILD_GROUP_INSTRUMENTED)
         self.copy_sources = True
+        self.patches = ['crypt_blowfish-makefile-cflags.patch']
+        self.patch_strip = 0
 
     def build(self, builder):
         log_prefix = builder.log_prefix(self)
         log_output(log_prefix, ['make', 'clean'])
-        log_output(log_prefix, ['make'])
+        if is_mac():
+            log_output(log_prefix, ['make', 'CFLAG_EXTRA="-mmacosx-version-min=10.14"'])
```

Comment:
I had an impression that we already had this kind of a flag somewhere. Can we deduplicate it so that we can specify the target macOS version once for the entire project?
Also if crypt_blowfish ignores the mechanism that we use to pass flags to other dependencies, could that cause problems? Perhaps we should add other flags here via the same mechanism?

Reply:
Good point. That flag, and all of the other standard flags are passed as CXXFLAGS. If I pass that value into blowfish build, it chokes because it is just using gcc, not gcc++. 

The "standard" flags we specify like "-fPIC -O2 -Wall" are already hard-coded in the makefile.  (omit-fram-pointer option is also hard-coded, though it specifies opposite of our standard option.)

In this case, it seems only this macosx flag is needed, but it makes sense to centralize it, in case the version ever needs to be modified.